plugins/modules/inventory_intent.py

```diff
@@ -1055,10 +1061,9 @@ def export_device_details(self):
         device_ips = self.config[0].get("ip_address", [])
 
         if not device_ips:
-            msg = "No Devices are given in the playbook so can't export device details"
             self.status = "failed"
-            self.msg = msg
-            self.log(msg)
+            self.msg = "No Devices are given in the playbook so can't export device details"
```

Comment:
Cannot export device details as no devices are specified in the playbook

Reply:
added.